.env_SAMPLE

```diff
@@ -175,4 +177,6 @@ export WATCHMAN_TOKENS=''
 # Project configuration - set up working state.
 ###############################################
 
-source activate-virtualenv.sh
+if [ ! -z $(command -v workon)]; then
```

Comment:
This needs a space before the `]` to work, otherwise you get this error:

`-bash: [: missing `]'`